### PR TITLE
Update XC to 9.2.0

### PR DESCRIPTION
**Pull request by cheemingli**
- Update to 9.2.0 XC package file names
- Added target XcSitecore and XcMssql to build image offline
- Added target to build redis image

Note: Bootstrap and initialization of the commerce engine is not done anymore to keep the build process offline.

### CODE REVIEW

File: xc/sitecore/InstallCommercePackages.ps1 [hunk 1]
```diff
@@ -150,12 +107,9 @@ $node.defaultShopName = $defaultShopName; `
 $xml.Save($pathToConfig);
 
 # Initialize the commerce engine
-$bearerToken = GetIdServerToken -userName $sitecoreUserName -password $sitecorePassword -urlIdentityServerGetToken "${identityServerUrl}connect/token"
+# TODO Enable
```
Comment (joostmeijles) [inline xc/sitecore/InstallCommercePackages.ps1]: TODO?

File: build/Build.Xc.cs [hunk 2]
```diff
@@ -346,10 +337,84 @@ partial class Build : NukeBuild
...
+                    $"COMMERCE_MA_PACKAGE={COMMERCE_MA_PACKAGE}",
+                    $"COMMERCE_XPROFILES_PACKAGE={COMMERCE_XPROFILES_PACKAGE}",
+                    $"COMMERCE_XANALYTICS_PACKAGE={COMMERCE_XANALYTICS_PACKAGE}",
+                    $"COMMERCE_SIF_PACKAGE={COMMERCE_SIF_PACKAGE}",
+                })
+            );
+        });
 
     Target Xc => _ => _
-        .DependsOn(XcCommerce, XcSitecoreMssql, XcSolr, XcXconnect, XcIdentity);
```
[the remark this answers is not in the document]
Comment (cheemingli) [reply build/Build.Xc.cs]: Not yet. We need to finish the upgrade for SXA and JSS target before we can remove them.

File: xc/commerce/Dockerfile [hunk 3]
```diff
@@ -178,23 +169,16 @@ RUN $CommerceServicesPathCollection = @('C:\\inetpub\\wwwroot\\CommerceAuthoring
...
-    $originalJson.EnvironmentName = $Env:ENVIRONMENT_NAME; `
-    $originalJson.ShopName = $Env:SHOP_NAME; `
-    $originalJson | ConvertTo-Json -Depth 100 -Compress | set-content $pathToJson;
-
 # Install plumber
 RUN Expand-Archive -Path "/Files/$Env:PLUMBER_FILE_NAME" -DestinationPath 'c:\\inetpub\\plumber'; `
     Import-Module -Name WebAdministration; `
     $iisApp = New-Item IIS:\Sites\Plumber -bindings @{protocol='http';bindingInformation='*:4000:*'} -physicalPath 'c:\inetpub\plumber'; `
-    $pathToJson = $(Join-Path -Path 'C:\\inetpub\\plumber' -ChildPath "static\\config.json"); `
+    $pathToJson = $(Join-Path -Path 'C:\\inetpub\\plumber' -ChildPath "config.json"); `
```
[the remark this answers is not in the document]
Comment (cheemingli) [reply xc/commerce/Dockerfile]: The latest (`1.1.3`) doesn't have the `static` directory.
Which version of Plumber do we support and where can I document it?

[the remark this answers is not in the document]
Comment (joostmeijles) [reply xc/commerce/Dockerfile]: Alright, we should maybe document it in the `plumber.zip` filename as we do for the other assets.

File: xc/sitecore/InstallCommercePackages.ps1 (hunk 1 above)
Comment (cheemingli) [reply xc/sitecore/InstallCommercePackages.ps1]: Removed TODO and the script in comments.

**Pull request merged**